pulsar/consumer_impl.go

```diff
@@ -489,6 +495,20 @@ func (c *consumer) ReconsumeLater(msg Message, delay time.Duration) {
 }
 
 func (c *consumer) Nack(msg Message) {
+	if c.options.EnableDefaultNackBackoffPolicy || c.options.NackBackoffPolicy != nil {
```

Comment:
Nit create a single variable at creation for checking
c.useBackoffPolicy or something like that